crates/ra_assists/src/handlers/replace_qualified_name_with_use.rs

```diff
@@ -459,6 +528,118 @@ use std::fmt::Debug;
 
 fn main() {
     Debug
+}
+    ",
+        );
+    }
+
+    #[test]
+    fn replaces_all_affected_paths() {
+        check_assist(
+            replace_qualified_name_with_use,
+            "
```

Comment:
We generally use `raw` literals here, if that's not required, because they get syntax higlighting.